napari/layers/_image_layer/view/controls.py

```diff
@@ -25,14 +25,10 @@ def __init__(self, layer):
         self.climSlider.rangeChanged.connect(self.clim_slider_changed)
 
     def clim_slider_changed(self, slidermin, slidermax):
-        from ..model import Image
-        for layer in self.layer.viewer.layers:
-            if isinstance(layer, Image) and layer.selected:
-                valmin, valmax = layer._clim_range
-                cmin = valmin+slidermin*(valmax-valmin)
-                cmax = valmin+slidermax*(valmax-valmin)
-                layer.clim = [cmin, cmax]
-                layer._qt_controls.clim_slider_update(None)
+        valmin, valmax = self.layer._clim_range
+        cmin = valmin+slidermin*(valmax-valmin)
+        cmax = valmin+slidermax*(valmax-valmin)
+        self.layer.clim = [cmin, cmax]
```

Comment:
the list brackets are probably unnecessary here

```suggestion
        self.layer.clim = cmin, cmax
```

Reply:
fixed